src/main/java/com/android/volley/toolbox/DiskBasedCacheUtility.java

```diff
@@ -37,6 +43,95 @@ static String getFilenameForKey(String key) {
         return localFilename;
     }
 
+    /** Returns a file object for the given cache key. */
+    public static File getFileForKey(String key, FileSupplier rootDirectorySupplier) {
+        return new File(rootDirectorySupplier.get(), getFilenameForKey(key));
+    }
+
+    static boolean wouldExceedCacheSize(long newTotalSize, int maxCacheSize) {
```

Comment:
nit: make `maxCacheSize` a long to be consistent with the total size

Reply:
The term `exceed` in this method name is misleading as implemented, but I think exceed is correct.

I'll collect comments on these two methods below.